middleware/admin/verify_attestation.py

```diff
@@ -45,6 +45,26 @@
                          "dad609"
 
 
+def validate_ui_message_header(ui_message):
+    minor_offset = len(UI_MESSAGE_HEADER) - 1
+    if ui_message[:minor_offset] != UI_MESSAGE_HEADER[:minor_offset]:
+        raise AdminError()
+    version_minor = ui_message[minor_offset]
+    # The minor version must be a single digit between 0 and 9
+    if version_minor < 48 or version_minor > 57:
+        raise AdminError()
+
+
+def validate_signer_message_header(signer_message):
+    minor_offset = len(SIGNER_MESSAGE_HEADER) - 1
+    if signer_message[:minor_offset] != SIGNER_MESSAGE_HEADER[:minor_offset]:
+        raise AdminError()
+    version_minor = signer_message[minor_offset]
+    # The minor version must be a single digit between 0 and 9
+    if version_minor < 48 or version_minor > 57:
+        raise AdminError()
+
+
```

Comment:
Why not use simple regular expressions for this? Like for example:

`re.compile(b"^HSM:UI:5.[0-9]$").match(b"HSM:UI:5.3") is not None`

Reply:
Sure, this sounds good, I'm making that change, thanks.